homework/04LessonFour.js

```diff
@@ -14,69 +14,91 @@ var numberArray = [1, 2, 3, 4, 5];
 
 // Problem 1:
 // Show what numberArray would look like if we called numberArray.pop()
-var numberArrayAfterPop = [];
+var numberArrayAfterPop = [1, 2, 3, 4];
+console.log(numberArrayAfterPop);
 
 // Problem 2:
 // Show what numberArray would look like if we called numberArray.shift()
-var numberArrayAfterShift = [];
+var numberArrayAfterShift = [2, 3, 4, 5];
+console.log(numberArrayAfterShift);
 
 // Problem 3:
 // Show what numberArray would look like if we called numberArray.push(6)
 var numberArrayAfterPush = [];
 
 // Problem 4:
 // Show what numberArray would look like if we called numberArray.unshift(0)
-var numberArrayAfterUnshift = [];
+var numberArrayAfterUnshift = [0, 1, 2, 3, 4, 5];
+console.log(numberArrayAfterUnshift);
 
 // Problem 5:
 // Use pop() and shift() to make the array bravestWarriors look like this:
 // ['Danny', 'Chris', 'Beth', 'Wallow']
 var bravestWarriors = ['Catbug', 'Danny', 'Chris', 'Beth', 'Wallow', 'Impossibear'];
+bravestWarriors.shift();
+bravestWarriors.pop();
+console.log(bravestWarriors);
 
 // Problem 6:
 // Use push() and unshift to make the array fruit look like this:
 // ['banana', 'kiwi', 'apple', 'orange', 'grapes', 'mango']
 var fruit = ['kiwi', 'apple', 'orange', 'grapes'];
+fruit.push('mango');
+fruit.unshift('banana');
+console.log(fruit);
 
 // ****
 // Objects (Associative Arrays)
 // ****
 
 // Problem 7:
 // Create an empty object
-var emptyObject;
+var emptyObject = {};
 
 // Problem 8:
 // Create an object with the following key, value pairs:
 // 1) emptyObject:  {}
 // 2) emptyArray:   []
 // 3) name:         'bob'
 // 4) number:       42
-var problemEightObject;
+var problemEightObject = {
+    emptyObject: {},
+    emptyArray: [],
+    name: 'bob',
+    number: 42
+}
+
+
+
+
 
 // Problem 9:
 // DON'T EDIT problemNineObject!!!!
 var problemNineObject = {
-  'fav food': 'pizza',
-  city:       'houston'
+    'fav food': 'pizza',
+    city: 'houston'
 };
 // update 'fav food' so that its value is tacos
 // update city so that its value is austin
+problemNineObject['fav food'] = 'tacos';
+problemNineObject['city'] = 'Austin';
+
 
 // Problem 10:
 // DON'T EDIT nestedObjects!!!!
 var nestedObjects = {
-  someNumbers: [1, 2, 3.14159, 4, 5, 6],
-  users: {
-    'fred astaire': {
-      hometown: 'Omaha'
-    },
-    'bob roberts': {
-      starring: 'John Cusack'
+    someNumbers: [1, 2, 3.14159, 4, 5, 6],
+    users: {
+        'fred astaire': {
+            hometown: 'Omaha'
+        },
+        'bob roberts': {
+            starring: 'John Cusack'
+        }
     }
-  }
 };
 // update the starring property to have the value 'Tim Robbins'
+[nestedObject['users']['bob roberts']['starring'] = 'Tim Robbins'
```

Comment:
Uh oh! this looks like a syntax error. The very first character is `[`; fix that typo and this should be correct!